azure/activity_logs_monitoring/index.js

```diff
@@ -132,11 +132,12 @@ class HTTPClient {
         this.httpOptions = {
             hostname: DD_HTTP_URL,
             port: DD_HTTP_PORT,
-            path: '/v1/input',
+            path: '/api/v2/logs',
             method: 'POST',
             headers: {
                 'Content-Type': 'application/json',
-                'DD-API-KEY': DD_API_KEY
+                'DD-API-KEY': DD_API_KEY,
+                'DD-SOURCE': DD_SOURCE
```

Comment:
I think this should be just "azure" always - I believe the source here is more for internal tracking of where logs are coming from vs the DD_SOURCE var is for the source in the log